suricata/update/main.py

```diff
@@ -1281,7 +1284,9 @@ def _main():
     suricata.update.net.set_user_agent_suricata_version(suricata_version.full)
 
     if args.subcommand:
-        if hasattr(args, "func"):
+        if args.subcommand == "check-versions" and hasattr(args, "func"):
+            return args.func(suricata_version)
```

Comment:
Maybe we should not alter the existing subcommand structure. Let it be having functions that do not take any arguments. The code is written in a manner that we can always get args or configuration values on demand.
Probably it would be nice to bring the entire block of code where `suricata_version` is evaluated into some other file.
@jasonish is it OK to move the version evaluation code in `version.py`?

Reply:
Leaving this as-is since extracting the version info from args seems to add a cyclic dependency in case the code is moved out.